public/content/lessons/2018/derivatives-and-transforms/index.mdx

```diff
@@ -1,10 +1,267 @@
+
+<Figure image="inf_13.png" video="inf_13.mp4" show="video" />
+
+Notice how in animating this with these dots representing some sample points, it doesn't take many iterations at all for most dots to clump in around $1.618$.
+
+Now, we know that $1.618$ and its little brother $-0.618$ stay fixed in place during each iteration of this process, but zoom in on the neighborhood around $1.618$. 
+
+<Figure image="inf_14.png" video="inf_14.mp4" show="video" />
+
+During the map, points in that region get contracted around $\varphi$, meaning the function $1 + \frac{1}{x}$ has a derivative with magnitude less than 1 at the input phi. In fact, this derivative works out to be around $-0.38$ at the input $\varphi$. So each repeated application scrunches that neighborhood smaller, like a gravitational pull towards $\varphi$.
+
+So now tell me what you think happens in a neighborhood of $\varphi$'s little brother. 
+
+<Figure image="inf_15.png" video="inf_15.mp4" show="video" />
+
+There, the derivative has a magnitude larger than $1$, so points near this fixed point are repelled away from it, by more than a factor of $2$ each iteration. Mathematicians would call this right value a “stable” fixed point, while the left one is an “unstable fixed point”. Something is considered stable if when you perturb it a little, it tends to come back towards where it started, rather than going away.
+
+<Figure image="inf_16.png" />
+
+So the stability of a fixed point is determined by whether the magnitude of its derivative is bigger or smaller than $1$, and this explains why $\varphi$ always shows up in the numerical play, and $\varphi$'s little brother does not. As to whether or not you want to consider $\varphi$'s little brother a valid value of our infinite fraction? Well, that's really up to you. Everything we just showed suggests that if you think of this expression as representing a limiting process, only phi makes sense. But maybe you think of it not as a limit, but as a static, purely algebraic object.
+
+<PiCreature
+  emotion="happy"
+  text="Either way, math is cool!"
+/>
+
+My point here not that understanding derivatives as measuring changes in density like this is somehow better than the graphical view on the whole. In fact, picturing an entire function this way, as opposed to a small window, can be clunky and impractical as compared to graphs. My point is that it deserves more of a mention in most introductory calculus courses, and can help make a student's understanding of the derivative a little more flexible.
+
+But as I mentioned, the real reason I'd recommend you carry this perspective with you as you learn new topics is not so much for how it will strengthen your understanding of single-variable calculus, but for what comes after. There are many topics typically taught in a college math department, which don't exactly have a reputation of being incredibly accessible. 
```

Comment:
You can maybe leave some links to videos/articles for where this goes, such as your video on complex derivatives, or the ones I did at Khan Academy on the Jacobian.